programs/github

```diff
@@ -134,24 +128,26 @@ if action == 'clone' then
 end
 
 if action == 'auth' then
+	local user, token = args[1], args[2]
+	if not user then return printError("No user specified.")
+
 	if flags.delete then
-		github.Auth.delete(flags.delete)
-		print(('Github authentication for %s successfully deleted.').format(flags.delete))
+		github.Auth.delete(user)
+		print(('Successfully deleted github token for user %s'):format(flags.delete))
```

Comment:
Actually, we can probably cut the world "Successfully` (here and below), given our terminals are narrow